bin/pbiviz.js

```diff
@@ -41,7 +41,7 @@ program
     .command('new [name]', 'Create a new visual')
     .command('info', 'Display info about the current visual')
     .command('start', 'Start the current visual')
-    .command('package', 'Package the current visual into a pbiviz file')
+    .command('package', 'Package the current visual into a pbiviz file. Use --resources to output static files into a resources folder. Use --no-pbiviz to not produce a pbiviz file. Use --no-minify to not minify pbiviz static files.')
```

Comment:
This information is available by running the following command: `pbiviz package --help`